### PR TITLE
Update the documentation

### PR DESCRIPTION
**Pull request by amontoison**
supersedes #85

https://juliasmoothoptimizers.github.io/MUMPS.jl/previews/PR96/

### CODE REVIEW

File: src/interface.jl [hunk 1]
```diff
@@ -338,32 +350,34 @@ function get_sol(mumps::Mumps{T}) where {T}
 end
 
 """
-    get_schur_complement!(S,mumps)
+    get_schur!(S,mumps)
```
Comment (dpo) [inline src/interface.jl:353]: Let's not change method names in a doc pr.

File: src/mumps_struc.jl [hunk 2]
```diff
@@ -16,7 +15,9 @@ const MUMPS_VERSION = "5.5.1"
 const MUMPS_VERSION_MAX_LEN = 30
 const DEFAULT_FORTRAN_COMMUNICATOR = -987654
 
-# mirror of structre in [sdcz]mumps_c.h
+"""
+mirror of structre in [sdcz]mumps_c.h
```
Comment (dpo) [inline src/mumps_struc.jl:19]:
```suggestion
Mirror of structre in `[sdcz]mumps_c.h`.
```

**Review by dpo — CHANGES_REQUESTED** [review]
This is great, thank you! Would you mind also simplifying the README to remove the repetitions with the docs? The README could simply point to the docs.

File: src/interface.jl (hunk 1 above)
Comment (amontoison) [reply src/interface.jl:353]: I need this modification to fix the documentation, I opened #97.

**Review by dpo — APPROVED** [review]

**Comment by dpo** [comment]
Thank you.

**Pull request merged**